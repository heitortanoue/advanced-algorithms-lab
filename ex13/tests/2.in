5
10
3
1 3 4
100
10
1 2 3 5 6 7 9 12 4 15
1000
10
1 2 3 5 6 7 9 12 4 15
900
10
1 2 3 5 6 7 9 12 4 15
990
10
1 2 3 5 6 7 9 12 4 15


