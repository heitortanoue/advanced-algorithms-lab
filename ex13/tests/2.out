3
8
68
60
66
